distribution/docs/src/main/resources/content/_configuring/hardening-user-access.adoc

```diff
@@ -11,6 +11,9 @@
 Once ${branding} is configured to use an external user (such as LDAP), remove the `users.properties` file from the `<INSTALL_HOME>/etc` directory.
 Use of a `users.properties` file should be limited to emergency recovery operations and replaced as soon as effectively possible.
 
+If SSH access to the Karaf shell is to be supported, edit the file `org.apache.karaf.shell.cfg` in the `<INSTALL_HOME>/etc` directory, changing the value
+of the `sshRealm` property from `karaf` to `ldap`.
+
 .Emergency Use of `users.properties` file
```

Comment:
Does the file `environment-hardening.adoc`need updating as well?
I editing the documentation with what I think it should say and attached it as a patch.
[doc_patch.patch.txt](https://github.com/codice/ddf/files/1366466/doc_patch.patch.txt)

Also is this warning in the same file still correct in a post security manager world?

```[WARNING]
====
Enabling SSH will expose your file system such that any user with access to your ${branding} shell will
have read/write access to all directories and files accessible to your installation user. It is not
recommended in a secure environment.
====
```

Reply:
Will apply the patch.

The second warning isn't quite correct, but is broadly correct in that whatever files your karaf install user can see through Java can be seen by someone shelling in. I'm not sure it's worth trying to parse that subtle difference here.